cli/js/form_data_test.ts

```diff
@@ -110,6 +114,7 @@ test(function formDataParamsArgumentsCheck(): void {
     let hasThrown = 0;
     let errMsg = "";
     try {
+      // @ts-ignore
```

Comment:
There should be a way to type this do you don't have to ignore.

Reply:
The issue here is that `formData[method]` would take 1 or 2 arguments, which isn't what we do here, so I'm not sure how to circumvent that here.

Reply:
Understood.